mysite.rb

```diff
@@ -0,0 +1,25 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require './lib/scrabble'
+
+class MySite < Sinatra::Base
```

Comment:
Great job with your routes. These are logical and using appropriate HTTP verbs for these scenarios.